jubakit/wrapper.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin
+from .classifier import Classifier, Config, Dataset
+
+
+class BaseJubatusClassifier(BaseEstimator, ClassifierMixin):
+  """
+  scikit-learn Wrapper for Jubatus Classifiers.
+  """
+  
+  def __init__(self, n_iter=1, shuffle=False, softmax=False, embedded=True, seed=None):
+    """
+    Creates a base class for Jubatus Classifiers"
+    """
+    self.softmax = softmax
+    self.n_iter = n_iter
+    self.shuffle = shuffle
+    self.embedded = embedded
+    self.seed = seed
+    self.fitted_ = False
```

Comment:
I see. As you proposed, using sklearn-style coding rule makes the code easier to understand for users with experience of sklearn.